collector/collector.go

```diff
@@ -81,15 +85,38 @@ func NewExporter(address string, logger log.Logger) Exporter {
 func (e Exporter) Describe(ch chan<- *prometheus.Desc) {
 }
 
+func (e Exporter) dial() (net.Conn, error, func()) {
+	if strings.ContainsAny(e.address, "/") {
```

Comment:
The preferred format for unix sockets is `unix://`. For example `unix:///run/chrony/chronyd.sock`. Then use `strings.HasPrefix()`.